staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go

```diff
@@ -266,12 +266,18 @@ func setVisitor(visitorConfig *driver.VisitorConfig, twin *common.Twin, dev *dri
 		return nil
 	}
 	klog.V(2).Infof("Convert type: %s, value: %s ", twin.Property.PProperty.DataType, twin.ObservedDesired.Value)
-	value, err := common.Convert(twin.Property.PProperty.DataType, twin.ObservedDesired.Value)
-	if err != nil {
-		klog.Errorf("Failed to convert value as %s : %v", twin.Property.PProperty.DataType, err)
-		return err
+	var value interface{}
+	if twin.ObservedDesired.Value != "" {
+		convertedValue, err := common.Convert(twin.Property.PProperty.DataType, twin.ObservedDesired.Value)
+		if err != nil {
+			klog.Errorf("Failed to convert value as %s : %v", twin.Property.PProperty.DataType, err)
+			return err
+		}
+		value = convertedValue
```

Comment:
```suggestion
		value, err := common.Convert(twin.Property.PProperty.DataType, twin.ObservedDesired.Value)
		if err != nil {
			klog.Errorf("Failed to convert value as %s : %v", twin.Property.PProperty.DataType, err)
			return err
		}
```

Reply:
done

Reply:
The scopes are different.

<img width="618" alt="image" src="https://github.com/user-attachments/assets/bb6936a3-944b-4ada-adac-3682f2e7a6fa">

Reply:
OK, I see